### DISCUSSION TITLE
Good first issue to contribute to?

### DISCUSSION THREAD

**Discussion by nico-mayora** [gorilla-devs/ferium#217] [General]
I'm interested on contributing to this project and was wondering what was a good first issue to get to know the codebase and such.

**Comment by theRookieCoder** [comment]
I think #196 and #179 might be decent starters. Keep in mind that most of the enhancements haven't been implemented for a reason :)

Don't worry if the code isn't perfect, I'll make changes anyways.

Good luck!
